Binary-Search/sqrt-of-int

```diff
@@ -0,0 +1,17 @@
+// https://www.interviewbit.com/problems/square-root-of-integer/
+int Solution::sqrt(int A) 
+{     
+  int begin = 0;
+  int end = A;
```

Comment:
Add the following corner case  
`if(A == 1 || A == 0) return A;`
Yours is giving an FPE for input  A = 1.